### DISCUSSION TITLE
More reliable Versioning possible?

### DISCUSSION THREAD

**Discussion by angstitc** [focus-shift/jollyday#498] [Q&A] [answered]
### More information

Hey Guys,

first of all great work here! 

I just would love to have a more reliable versioning. As a developer I would not expect breaking changes in minor or patch version updates but in major ones. Thats just the hint I want to give or more the question, if this is applicable for you in the future?

Thanks and best regards,
Fabian

**Comment by derTobsch** [comment, marked as answer]
Hey @angstitc,

we are in an "early" stage of the development of this library and are using semantic versioning. I don't know how familiar you are with semantic versioning, but the 0.x.x versions are used for development. That means breaking changes may occur any time, see https://www.baeldung.com/cs/semantic-versioning#development-semver or https://semver.org/#spec-item-4.

I think there will maybe be some development versions coming until 1.0.0. Maybe something around 0.3x.0 and then I will release the 1.0.0. I wanted to take some deeper looks at the API and make it consistent, renaming the holiday types #462 and look for further improvements in consistency. 

Hope that helps.
